Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "windows-service"
-version = "0.6.0"
+version = "0.6.2"
```

Comment:
Please don't bump the verison in feature PRs. We'll do that prior to release in a manner suitable for the collective changes since last release.